tests/14-policy-enforcement-docker.sh

```diff
@@ -87,7 +87,7 @@ function import_sample_policy {
 [{
      "endpointSelector": {"matchLabels":{"k8s:id":"app3"}},
      "egress": [{
-         "toCIDR": [ { "ip": "9.9.9.9/32" } ]
+         "toCIDR": [ "9.9.9.9/32" ]
```

Comment:
This can be squashed with ab2dd4ead4fb30ab4baeb5acd32dd13b19d80c8e (2 commits before)